usefulUtilities/deployNewScripts/README.md

```diff
@@ -0,0 +1,24 @@
+# Deploy New Scripts App
+This app lets you quickly replace old client and server scripts on entities with new scripts.
+
+# Usage Instructions

```

Comment:
Should we mention that they may need to perform the swap in multiple places to 'find' all the entities that need to change? Is this even still necessary?

Reply:
I don't think this is a necessary thing to add to the README, although I do understand that what you're saying is true. If users of this app don't understand how to use it, I'll have to change something later (whether that's the app or the README).